js/repl/loadScript.js

```diff
@@ -2,8 +2,36 @@
 
 import type { LoadScriptCallback } from "./types";
 
-export default function loadScript(url: string, callback: LoadScriptCallback) {
-  const script = document.createElement("script");
+export default function loadScript(
+  url: string,
+  callback: LoadScriptCallback,
+  target: ?HTMLIFrameElement
+) {
+  if (target != null) {
+    const targetDocument = target.contentDocument;
+
+    // If we're loading this script into an iframe, wait for it to load,
+    // Otherwise the script tag we insert will be removed on-load.
+    if (targetDocument.readyState !== "complete") {
+      target.addEventListener("load", () => {
+        // Use the newly-loaded document.
+        // $FlowFixMe We know target isn't null at this point
```

Comment:
Might be able to pull it into a `const` variable to avoid the FlowFixMe:

```
export default function loadScript(
  url: string,
  callback: LoadScriptCallback,
  maybeTarget: ?HTMLIFrameElement
) {
  const target = maybeTarget;
  if (target != null) {
    // ...
```

Reply:
Cool. I'll change to use a local const then. 😄